doc/method/results.md

```diff
@@ -1,35 +1,9 @@
 
-## Contributions of source taxonomies to the assembly
```

Comment:
It's moved, not removed.  Scripted generation is now part of document build process.